### DISCUSSION TITLE
Bakteri deneyi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#229] [Soru Cevap]
Bir deney ortamında bulunan bir bakteri kolonisi 4 saatte bir gözlemlenmekte ve bakteri sayısının düzenli olarak bir önceki gözleme oranla %25 arttığı saptanmaktadır. Bakteri sayısının deney başlangıcındaki sayının 3 katını aşması, deney başladıktan kaç saat sonra ilk olarak gözlemlenecektir?

**Comment by Burak2210** [comment]
```
bakteri_sayisi = 1
saat = 0
while bakteri_sayisi < 3:
    bakteri_sayisi =  bakteri_sayisi + (bakteri_sayisi/4)
    saat = saat + 4
print(saat)
```
